src/Uno.Toolkit.RuntimeTests/Tests/NavigationBarTests.cs

```diff
@@ -383,5 +390,40 @@ public NavBarTestPage()
 		}
 #endif
 	}
+
+#if __IOS__
+	public static class NavigationBarTestHelper
+	{
+		public static UINavigationBar? GetNativeNavBar(this NavigationBar? navBar)
```

Comment:
while not really that much needed now, with #nullable in place
it would be nice to rename method that _may_ returns something from GetXyz to FindXyz
to highlight the fact that somtime you may not get what you expected